UnitySDK/Assets/ML-Agents/Examples/SharedAssets/Scripts/RayPerception3D.cs

```diff
@@ -10,9 +11,6 @@ namespace MLAgents
     /// </summary>
     public class RayPerception3D : RayPerception
     {
-        RaycastHit m_Hit;
-        float[] m_SubList;
```

Comment:
Same comment as above. Also RayCast hit is a struct so we don't need to keep it around (I think, correct me if I'm wrong)